<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmBase64HelperClass</class>
 <widget class="QWidget" name="FrmBase64HelperClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FrmBase64Helper</string>
  </property>
  <widget class="QTextEdit" name="textEditBase64">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>86</y>
     <width>491</width>
     <height>161</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 241, 83);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>313</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditInputData">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOpenFile">
        <property name="text">
         <string>打开文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonImageToBase64">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>图片转base64</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonTextToBase64">
        <property name="text">
         <string>文字转base64</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEditText">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonClearDate">
        <property name="text">
         <string>清空数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBase64ToImage">
        <property name="text">
         <string>base64转图片</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBase64ToText">
        <property name="text">
         <string>base64转文字</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelCostTimeTip">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FrmBase64Helper.qrc"/>
 </resources>
 <connections/>
</ui>
